docs/1.getting-started/8.server.md

```diff
@@ -0,0 +1,92 @@
+---
+
+# Server
+
+The ability to create, within your app, server endpoints is one of the new core feature of Nuxt 3. It allows you to **build full-stack applications**, fetch data from a database or another server, create an API or even generate static server-side content like a sitemap or a RSS feed, from a single codebase.
+
+::alert{type="info"}
+For more informations about usage of the `~/server` directory, please visit the [server directory structure section](/docs/guide/directory-structure/server).
+::
+
+## Powered by Nitro
+
+Nuxt 3 server-side features are powered by [Nitro](https://github.com/unjs/nitro), a [UnJS](https://unjs.io) project, while the client side features are powered by [Vite](https://vitejs.dev) or [webpack](https://webpack.js.org).
+
+::alert{type="info"}
+Note that Nitro is framework agnostic and can be used without Nuxt.
+::
+
+Using [Nitro](https://github.com/unjs/nitro) give Nuxt superpowers:
+
+- Full control of the server-side part of your app
+- Deploy your app anywhere
+- Hybrid rendering for your app
+
+## Full control of the server-side part of your app
+
+With Nitro, you can easily manage the server-side of your Nuxt 3 app; from endpoints to middleware.
+
+Both endpoints and middleware can be defined like this:
+
+```ts [server/{routes,middleware}/index.ts]
+export default defineEventHandler({
+  // ... Do whatever you want here
+})
+```
+
+And you can directly return `text`, `json`, `html` or even a `stream`.
+
+Out-of-the-box, Nitro supports **hot module replacement** and **auto-import** like the other parts of your Nuxt application.
+
+::alert{type="info"}
+Learn more about endpoints and middlewares in [the server directory structure section](/docs/guide/directory-structure/server).
+::
+
+## Deploy your app anywhere
+
+[`Nitro`](https://github.com/unjs/nitro) offer the ability to deploy your Nuxt 3 anywhere, from a bare metal server to the edge network, with a start time near to 1ms. That's fast!
+
+Today, [`Nitro`](https://github.com/unjs/nitro) offers more than 15 presets to build your Nuxt 3 app for different cloud providers and servers, including:
+
+- [Cloudflare Workers](https://workers.cloudflare.com/)
+- [Netlify Functions](https://www.netlify.com/products/functions/)
+- [Vercel Edge Network](https://vercel.com/docs/edge-network/introduction)
+- [AWS Lambda](https://aws.amazon.com/lambda/)
+- [IIS](https://www.iis.net/)
+
+Or for other runtime:
+
+- [Deno](https://deno.land/)
+- [Bun](https://bun.sh/)
+
+::alert{type="info"}
+Learn more about deployment in [the related section](https://nuxt.com/docs/getting-started/deployment).
+::
+
+## Hybrid rendering for your app
+
+Need for both static and dynamic pages in your Nuxt 3 app? Nitro covers your back!
+
+Nitro has a powerful feature called `routeRules` which allow you to define a set of rules to customize the routes rendering mode of your Nuxt 3 app.
```

Comment:
We need to refine this later once we have universal route rules, nitro is not only place that benefits nuxt users with routeRules